### PR TITLE
feat(relay): Bump version to 0.8.20

### PR DESCRIPTION
**Pull request by jjbayer**
Use the newest relay library version.

Fixes https://github.com/getsentry/team-ingest/issues/107.

### CODE REVIEW

**Review by olksdr — APPROVED** [review]
:shipit:

File: fixtures/emails/n1_api_call.txt [hunk 1]
```diff
@@ -16,7 +16,7 @@ Tags
 
 * browser = Chrome 108.0.0
 * browser.name = Chrome
-* client_os = Mac OS X 10.15.7
+* client_os = Mac OS X &gt;=10.15.7
```
Comment (iker-barriocanal) [inline fixtures/emails/n1_api_call.txt:19]: Is this change related to the dependency bump?

**Review by iker-barriocanal — APPROVED** [review]

File: fixtures/emails/n1_api_call.txt (hunk 1 above)
Comment (jjbayer) [reply fixtures/emails/n1_api_call.txt:19]: Yes, this PR specifically: https://github.com/getsentry/relay/pull/1945
